modules/gdnative/gdnative_api.json

```diff
@@ -5753,6 +5753,14 @@
         "arguments": [
           ["const godot_string *", "p_message"]
         ]
+      },
+      {
+        "name": "godot_profiling_add_data",
```

Comment:
This should be part of the NativeScript 1.1 struct and not the GDNative core API

Reply:
I haven't checked myself NativeScript 1.1 yet. I should remove this gdnative_api.json changes and add it where?

Reply:
It's the right JSON file but the wrong place where you put the new information.

The JSON file is made up out of the core API and a list of extensions. NativeScript is GDNative extensions, and NativeScript 1.1 is a set of additional functions for NativeScript.

The place where you should put this is around here

https://github.com/godotengine/godot/blob/238b70e2db7e376ce8b627bbba5770535270da30/modules/gdnative/gdnative_api.json#L5879

Reply:
Thanks! Just moved it to where you pointed out.